### ISSUE TITLE
Missing mount of /tmp/.ssh folder

### ISSUE THREAD

**Issue by jost-kuenzel** [frgmt/vagrant-invade#2]
Hey frgmt. First thanks for this nice plugin. 
Migrating from the old invade system I discovered that currently the /tmp/.ssh is not mounted. And I cannot define the mount in the 'synced_folder' as the key 'nfs' is already defined for the project folder.
/tmp/.ssh is needed for https://gitlab.innogames.de/backend/puppet-ssh.
A suggestion I have is to use the key as a name with meaning rather that as the filesystem type.
So instead of e.g.

``` yml
    synced_folder:
      nfs:
        name: 'www'
        path: '/www'
        source: '..'
        mount_options:
          - nolock
```

it could be

``` yml
    synced_folder:
      www:
        type: 'nfs'
        path: '/www'
        source: '..'
        mount_options:
          - nolock
```

I'd be happy to hear your opinion on this.

**Comment by lennartstein** [comment]
hey jost, 
I'm sorry you have issues with defining a second synced folder for getting a copy of your ssh-folder on your guest machine. I will check this out and do some tests.
What you recommended was my first intention to build the synced_folder path too, but I came up with the idea of using the key for the type because I thought it would be smarter. Give me some time to check it out.

**Comment by lennartstein** [comment]
Fixed with version 0.4.6
Using hash key not as type anymore. Keep in mind to use type option from now on in defining synced folders AND provisioner

**Comment by jost-kuenzel** [comment]
Thank you, Lennart – It works perfectly!
